src/component/logoutButton/logoutButton.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+
+class LogoutButton extends Component {
+  logout = () => {
+    localStorage.removeItem("id_token");
+  };
+
+  render() {
+    return (
+      <Link to={"/"}>
+        <button onClick={this.logout}>Logout</button>;
+      </Link>
+    );
+  }
+}
+
+export default LogoutButton;
```

Comment:
Nice file 👍 

Maybe I would have chosen the following file naming convention though:

`src/components/buttons/logout.js`

Then if `logout.js` is divided up into different parts, then look to change it to:

`src/components/buttons/logout/index.js` (or `logout.js`), then keep other files in the `logout` `dir`

Reply:
Oh awesome, thanks! That makes a lot of sense, so I've changed it now.